MSAL/test/app/ios/MSALTestAppAuthorityViewController.h

```diff
@@ -28,7 +28,13 @@
 #import "MSALTestAppSettingViewController.h"
 
 @interface MSALTestAppAuthorityViewController : MSALTestAppSettingViewController
+{
+    NSMutableArray *_authorities;
+    NSMutableArray *_savedAuthorities;
+}
 
 + (instancetype)sharedController;
+- (instancetype)initWithAuthorities:(NSMutableArray *)authorities
+                    addUserDefaultsForKey:(NSString *)userDefaultsKey;
```

Comment:
nit: addUserDefaultsForKey is not very intuitive, let's maybe call it "savedAuthoritiesKey" or "keyForSavedAuthorities"...

Reply:
done